test_scripts/API/AddCommand/018_ATF_VR_IsReady_no_resp_AddCommand_VR_AddCommand_UNSUP_RES.lua

```diff
@@ -0,0 +1,154 @@
+  testCasesForVR_IsReady.InitHMI_onReady_without_VR_IsReady(self, 1)
+  EXPECT_HMICALL("VR.IsReady")
+  -- Do not send HMI response of VR.IsReady
+end
+
+function Test:Precondition_connectMobile()
+  self:connectMobile()
+end
+
+function Test:Precondition_StartSession()
+  self.mobileSession = mobile_session.MobileSession(self, self.mobileConnection)
+  self.mobileSession:StartService(7)
+end
+
+commonSteps:RegisterAppInterface("Precondition_RegisterAppInterface")
+
+function Test:Precondition_ActivationApp()
+  local request_id = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
+  EXPECT_HMIRESPONSE(request_id)
+  :Do(function(_,data)
+    if (data.result.isSDLAllowed ~= true) then
+      local request_id1 = self.hmiConnection:SendRequest("SDL.GetUserFriendlyMessage", {language = "EN-US", messageCodes = {"DataConsent"}})
+      EXPECT_HMIRESPONSE(request_id1)
+      :Do(function(_,_)
+        self.hmiConnection:SendNotification("SDL.OnAllowSDLFunctionality", {allowed = true, source = "GUI", device = {id = config.deviceMAC, name = ServerAddress}})
+        EXPECT_HMICALL("BasicCommunication.ActivateApp")
+        :Do(function(_,data1)
+          self.hmiConnection:SendResponse(data1.id,"BasicCommunication.ActivateApp", "SUCCESS", {})
+        end)
+      end)
+    end
+  end)
+  EXPECT_NOTIFICATION("OnHMIStatus", {hmiLevel = "FULL", systemContext = "MAIN"})
+end
+
+commonSteps:PutFile("Precondition_PutFile", "icon.png")
+
+--[[ Test ]]
+commonFunctions:newTestCasesGroup("Test")
+
+function Test:TestStep_AddCommand_UI_warnings()
+  local cid = self.mobileSession:SendRPC("AddCommand",
+  {
+    cmdID = 11,
+    menuParams =
+    {
+      parentID = 0,
+      position = 0,
+      menuName ="Commandpositive"
+    },
+    vrCommands =
+    {
+      "VRCommandonepositive",
+      "VRCommandonepositivedouble"
+    },
+    cmdIcon =
+    {
+      value ="icon.png",
+      imageType ="DYNAMIC"
+    }
+  })
+
+  EXPECT_HMICALL("UI.AddCommand",
+  {
+    cmdID = 11,
+    menuParams = { parentID = 0, position = 0, menuName ="Commandpositive" }
+  })
+  :ValidIf(function(_,data)
+    local value_Icon = storagePath .. "icon.png"
+    if (string.match(data.params.cmdIcon.value, "%S*" .. "("..string.sub(storagePath, 2).."icon.png)" .. "$") == nil ) then
+      print("\27[31m value of cmdIcon is WRONG. Expected: ~".. value_Icon .. "; Real: " .. data.params.cmdIcon.value .. "\27[0m")
+      return false
+    else
+      return true
+    end
+  end)
+  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, "UI.AddCommand", "WARNINGS", {}) end)
+
+  EXPECT_HMICALL("VR.AddCommand",
+  {
+    cmdID = 11,
+    vrCommands = { "VRCommandonepositive", "VRCommandonepositivedouble"},
+    type = "Command"
+  })
+  :Do(function(_,data) self.hmiConnection:SendResponse(data.id, "UI.AddCommand", "UNSUPPORTED_RESOURCE", {info = "unsupported resource"}) end)
```

Comment:
@istoimenova , Please see update in commit 36d7d4238d3cf494d255c705f9540101c11a9d98